libraries/botframework-streaming-extensions/src/Models/StreamDescription.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * @module botframework-streaming-extensions
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+export class StreamDescription {

```

Comment:
I'm thinking this over. The idea behind the description was to avoid needing the streams to exist as a prerequisite to sending a request/response with information about them, but I'm not sure how useful that really is.